root/utility/tableColumns.js

```diff
@@ -282,14 +293,15 @@ export function defineInstrumentUsageColumn(
 }
 
 export function defineNameColumn<T: CoreEntityT | CollectionT>(
-  title: string,
-  order?: string = '',
-  sortable?: boolean = false,
-  descriptive?: boolean = true,
+  props: {
+    ...OrderableProps,
+    +descriptive: boolean,
+    +title: string,
+  },
 ): ColumnOptions<T, string> {
   return {
     Cell: ({row: {original}}) => (
-      descriptive
+      props.descriptive
```

Comment:
```suggestion
    const descriptive = props.hasOwnProperty('descriptive') ? descriptive : true;
    return {
      Cell: ({row: {original}}) => (
        descriptive
```

Reply:
Is this better though?

I feel defaulting things to true is confusing, so I thought specifying when we do want descriptive linking seemed more clear.

On the other hand, it's true that we want descriptive linking in most cases, so maybe the clearest option of them all would be change the condition from "descriptive" to "simpleLink" or something like that, and then only pass it when we specifically want to deviate from the (descriptive) norm?

Reply:
Default value can be either `true` or `false`, in either case that will be far better than requiring to set it for the vast majority of calls. “simpleLink” is simply not as descriptive as its negation “descriptive”, but there might be a clearer wording.

Reply:
Ok, since it seems it's just me who finds this kinda weird, then let's do it :)